native/monogame/include/api_common.h

```diff
@@ -19,8 +19,30 @@ typedef double               mgdouble;
 
 #define MG_FIELD_OFFSET(off, type, name) struct { char _pad_##name[off]; type name; }
 
+#if defined(_WIN32) || defined(__CYGWIN__)
```

Comment:
Not sure how these changes in the export will affect the consoles... will have to test it later and fix things.  It however worked before.